libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -453,12 +455,41 @@ public void appendMediaFile(final MediaFile mediaFile, final String mediaUrl, Im
                 // TODO: insert video
                 ToastUtils.showToast(getActivity(), R.string.media_insert_unimplemented);
             } else {
-                // TODO: insert image
-                ToastUtils.showToast(getActivity(), R.string.media_insert_unimplemented);
+                imageLoader.get(mediaUrl, new ImageLoader.ImageListener() {
+                    @Override
+                    public void onErrorResponse(VolleyError error) {
+                    }
+
+                    @Override
+                    public void onResponse(ImageLoader.ImageContainer container, boolean isImmediate) {
+                        Bitmap downloadedBitmap = container.getBitmap();
+
+                        if (downloadedBitmap == null) {
+                            // No bitmap downloaded from server.
+                            return;
+                        }
+
+                        AztecAttributes attrs = new AztecAttributes();
+                        attrs.setValue("src", mediaUrl);
+
+                        if (downloadedBitmap.getWidth() < MIN_BITMAP_WIDTH) {
```

Comment:
The `200` value was taken from [`LegacyEditorFragment.MIN_THUMBNAIL_WIDTH`](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/LegacyEditorFragment.java#L79).  I updated both the minimum size to be 48dp according to the touch target [guidelines](https://material.io/guidelines/usability/accessibility.html#accessibility-style) with 749b046 and error message to specify the small condition with bb552e8.